core-java-modules/ParsingDates/src/main/java/com/parse/parsingdates/SimpleDateTimeFormater.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.parse.parsingdates;
+
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.format.DateTimeParseException;
+
+public class SimpleDateTimeFormater {
+
+    public boolean parseDate(String date) {
+        DateTimeFormatterBuilder dateTimeFormatterBuilder = new DateTimeFormatterBuilder()
+                .append(DateTimeFormatter.ofPattern("[MM/dd/yyyy]" + "[dd-MM-yyyy]" + "[yyyy-MM-dd]"));
+
+        DateTimeFormatter dateTimeFormatter = dateTimeFormatterBuilder.toFormatter();
+
+        try {
+            dateTimeFormatter.parse(date);
+            return true;
+        } catch (DateTimeParseException e) {
+            return false;
+        }

```

Comment:
Does it make sense to coerce this down to a boolean, or is it better to just return the parsed date so that the test can do what it needs to do?

In tests, if we expect parsing to fail, we can use `assertThatThrownBy` to watch the function under test and catch the thrown exception.